src/Sinks/Json/MultilineJsonSink.cs

```diff
@@ -0,0 +1,240 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Akka;
+using Akka.Actor;
+using Akka.Event;
+using Akka.Streams;
+using Akka.Streams.Stage;
+using Arcane.Framework.Sinks.Parquet;
+using Parquet.Data;
+using Snd.Sdk.Storage.Base;
+using Snd.Sdk.Storage.Models;
+using Snd.Sdk.Tasks;
+
+namespace Arcane.Framework.Sinks.Json;
+
+/// <summary>
+/// Sink that writes a list of JSON objects to a file divided by new lines.
```

Comment:
Sink that writes incoming lists of elements into a file per list, each element as a JSON-serialised line in the file, on a BLOB storage.
Also optionally drops a schema for the output in a form of an empty PARQUET file.

Reply:
Added this to docstring